R/hp_datatree.R

```diff
@@ -52,9 +52,9 @@ dt_collect <- function(x) {
 #'     name: Toulouse
 #'     job: Systems Engineer
 #'     species: Cat
-#'     jojo:
-#'       name: Jojo
-#'       job: Python Programmer
+#'     toulouse:
```

Comment:
get jojo back!!! 😭

Reply:
Wait I thought I brought jojo back...

Reply:
ya not sure how it got reverted back to toulous...i'll merge for now. can fix later.